CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Requests with empty body and application/json Content-Type Header will now
   return 400 error instead of 500 error.
   [cyberark/conjur#1968](https://github.com/cyberark/conjur/issues/1968)
+- `resources` requests with non numeric delimiter (limit or offset) will now
```

Comment:
it is more than that. Your change will not count non numeric input as valid input because before offset=somestring returned a whole list previously